tests/experimental/datasources/integration/conftest.py

```diff
@@ -154,23 +155,39 @@ def multibatch_spark_data(
     return context, spark_ds, asset, batch_request
 
 
-@pytest.fixture(params=[pandas_data, sql_data, spark_data])
+@pytest.fixture(
+    params=[
+        pandas_data,
+        sql_data,
+        pytest.param(
+            spark_data,
+            marks=pytest.mark.skipif(
+                not is_library_loadable(library_name="pyspark"),
```

Comment:
I don't think this is the right skip condition. We want to skip unless `--spark` is passed and want to run these if `--spark` is passed even if pyspark is not installed. That is because we don't want to think these run and pass when we pass `--spark` but they are just skipped instead.

Reply:
@billdirks I need some more helping getting this right.  Please correct any of the following, or confirm that this is on target:

1) This is about integration tests.
2) If `--spark` is not passed, we want to skip.
3) If `--spark` is passed, then we want to run, whether or not `pyspark` is installed.
4) If `--spark` is passed and `pyspark` is not installed, then test will error (it will be unable to run), and this is the right behavior, because it will direct us to update our `pip install` requirements for these integration tests.

Thank you.